README.md

```diff
@@ -72,12 +72,21 @@ The following fields will determine how the Ciao Web UI will beconfigured.
 
 #### Building application
 
-In order to build application the 'install.sh' must be executed. This script will intsall dependencies using the npm package manager and also build minified JS scripts used with browser compatibility.
-Note: this process is only necessary for development as minified scripts and dependencies are already provided by the application.
+In order to build the application a Makefile is provided. Use make 'TARGET' to build Ciao-Webui. When running either 'install' or 'install-dev' targets. Note that if not set, the environment variable 'NODE_ENV' will be set to 'production' by default.
+
+###### make install
+Use 'make' or 'make install' in order to build ciao-webui application with latest 'production' ready code.
+
```

Comment:
Hi @markdryan yes, the difference is that make install does not recompile js code, make install-dev does, I'll reword to make it more understandable. Thanks